### ISSUE TITLE
clarification on dotty lsp plugin

### ISSUE THREAD

**Issue by bbarker** [gtache/intellij-lsp#23]
I would really like to use the dotty plugin; it looks like it isn't currently published to a plugin repository for intellij. Is the recommendation to build it from source oneself currently? What advantages would this have, if any, over just using the standard intellij-scala plugin with intellij-lsp configured to talk to dotty's language server?

**Comment by smarter** [comment]
Right now I don't think there's any, but eventually the dotty plugin should provide syntax highlighting.

**Comment by gtache** [comment]
Indeed, there isn't any at the moment, I tried cutting through it once to only keep the syntax highlighting but it's quite complicated.
